crt_portal/cts_forms/locale/es/LC_MESSAGES/django.po

```diff
@@ -2614,6 +2627,25 @@ msgstr "Revise su informe"
 msgid "Please select if any that apply to your situation (optional)"
 msgstr "Favor de elegir esta opción si"
 
+#~ msgid "Have you or someone you know experienced unlawful discrimination?"
+#~ msgstr ""
+#~ "¿Usted o alguien que conoce ha sido víctima de discriminación ilícita?"
+
+#~ msgid "The Civil Rights Division may be able to help."
+#~ msgstr "Es posible que la División de Derechos Civiles podrá ayudarle."
+
+#~ msgid ""
+#~ "We enforce federal laws that protect you from discrimination based on "
+#~ "your race, color, national origin, disability status, sex, religion, "
+#~ "familial status, and more."
+#~ msgstr ""
+#~ "Nosotros hacemos cumplir las leyes federales que protegen a las personas "
+#~ "de la discriminación por motivos de su raza, color de piel, origen "
+#~ "nacional, discapacidad, género, religión o estatus familiar."
+
+#~ msgid "Common civil rights violations"
+#~ msgstr "Vulneraciones de derechos civiles comunes"
```

Comment:
should we remove instead of commenting out?

Reply:
Honestly not sure, this is the default behavior of `makemessages` when a translated string is removed from then code, `makemessages` comments it out in the `Django.po` file as opposed to deleting it.  Thoughts?

Reply:
Totally fine with default behavior. Less merge conflicts down the road if we leave vs delete, I suspect. Thanks for explaining!